feeds/combiner.go

```diff
@@ -172,19 +173,22 @@ func (cf *combinerFeed) forEachAlert(alertHandlers []cfHandler) error {
 		lowerBound := DefaultLookbackPeriod
 		upperBound := int64(0)
 
+		errGrp, ctx := errgroup.WithContext(cf.ctx)
+
+		subscriptions := cf.Subscriptions()
 		// Query all subscriptions and process alerts
-		for _, subscription := range cf.Subscriptions() {
-			logger = logger.WithFields(
-				log.Fields{
-					"subscriberBotId": subscription.Subscriber.BotID,
-					"subscribedBotId": subscription.Subscription.BotId,
+		for i := range subscriptions {
+			subscription := subscriptions[i]
+			errGrp.Go(
```

Comment:
If one of these request return an error, does the implementation in `errgroup` abort all requests?

Reply:
yes, and that can cause a problem when we enable data fees. replaced it with `sync.WaitGroup`